app/models/agents/qpx_agent.rb

```diff
@@ -0,0 +1,91 @@
+module Agents
+  class QpxAgent < Agent
+
+    cannot_receive_events!
+    default_schedule "every_12h"
+
+
+    description <<-MD
+      The QpxExpressAgent will tell you the minimum airline prices between a pair of cities. The api limit is 50 requests/day.
```

Comment:
The QpxExpress API is basically google flights, isn't it? Would it make sense to name the agent `GoogleFlightsAgent` and mention the qpx api in the description?